python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -220,7 +221,6 @@ def quantize(self, model,  # remove the type requirement for type checking
                  accuracy_criterion: dict = {'relative': 0.99, 'higher_is_better': True},
                  approach='static',
                  method='fx',
-                 backend='inc',
```

Comment:
also need to remove the python doc item below

Reply:
removed